src/Codeception/Module/ZF2.php

```diff
@@ -71,14 +73,25 @@ public function _initialize()
         if (isset($applicationConfig['module_listener_options']['config_cache_enabled'])) {
             $applicationConfig['module_listener_options']['config_cache_enabled'] = false;
         }
-        Console::overrideIsConsole(false);
-    }
+        Console::overrideIsConsole(false); 
+
+        $this->application = Application::init($this->applicationConfig);  
+        $this->serviceLocator = $this->application->getServiceManager();
+        if($this->serviceLocator->has('Doctrine\ORM\EntityManager')) {
+            $this->entityManager =  $this->serviceLocator->get('Doctrine\ORM\EntityManager');     
+        }          
+    }  
 
     public function _before(TestCase $test)
-    {
-        $this->application = Application::init($this->applicationConfig);
+    {      
+        $this->application = Application::init($this->applicationConfig);               
+        $this->serviceLocator = $this->application->getServiceManager();
+        if($this->serviceLocator->has('Doctrine\ORM\EntityManager')) {
+            $this->entityManager =  $this->serviceLocator->get('Doctrine\ORM\EntityManager');     
```

Comment:
why do you ever need this `entityManager` to be set before each test? Is that used somehow anywhere? I don't see it.